tests/test_integration.py

```diff
@@ -83,6 +83,25 @@ def test_alternate_executable_support(self):
                     self.assertEqual(status, 200)
                     self.assertEqual(json.loads(response)["stdout"], expected)
 
+    def test_gil_status(self):
+        """Test no-gil builds actually don't have a gil."""
+        with run_gunicorn():
+            get_gil_status = {
+                "input": "import sysconfig; print(sysconfig.get_config_var('Py_GIL_DISABLED'))"
+            }
+            cases = [
+                ("3.13", "0\n"),
+                ("3.13t", "1\n"),
+            ]
+            for version, expected in cases:
+                with self.subTest(version=version, expected=expected):
+                    response, status = snekbox_request(
+                        get_gil_status
+                        | {"executable_path": f"/snekbin/python/{version}/bin/python"}
```

Comment:
Nit: can we make this more readable by creating the dict on a separate line and saving it in a variable?

Reply:
https://github.com/python-discord/snekbox/pull/218/commits/7551599a8a085d27b48a233032fe58fcd738cb80